### PR TITLE
Fix some issues related to information on not publishing swpro on app…

### PR DESCRIPTION
**Pull request by PranavPurwar**
… stores

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,6 @@ Want to chat with us, discuss changes, or just hang out? We have a Discord serve
 
 This mod was not created for any harmful purposes, such as harming Sketchware; quite the opposite, actually. It was made to keep Sketchware alive by the community for the community. Please use it at your own discretion and consider becoming a Patreon backer to support the developers. Unfortunately, other ways to support them are not working anymore, so Patreon is the only available option currently. You can find their Patreon page [here](https://www.patreon.com/sketchware).
 
-We do NOT permit the modification of this repository then post it on Play Store or any other app store as a new app. Keep in mind that this project is still a mod. Competing with the original company using a mod is considered illegal, and we strongly discourage such behavior.
+We do NOT permit publishing Sketchware Pro on Play Store or on any other app store. Keep in mind that this project is still a mod. Unauthorised modding of apps is considered illegal and we discourage such behavior.
```
Comment (ilyassesalama) [inline README.md:88]: > We do NOT permit publishing Sketchware Pro...
So by this change, you mean anyone can change the app name to something else and publish it as long as it's not named Sketchware Pro?

Comment (PranavPurwar) [reply README.md]: check now

Comment (ilyassesalama) [reply README.md]: Looks good. Also, why `unauthorised` and not `unauthorized`? UK moment.

Comment (JavkhlanK) [reply README.md]: the readme uses US English so stick to that please

Comment (PranavPurwar) [reply README.md]: fixed

**Pull request merged**